### ISSUE TITLE
Bug report: Buttons not responding on drum kit

### ISSUE THREAD

**Issue by silverwavetidaldragon** [utkarsh1311/web-dev-practice#4]
I can see the on-screen keys do an appropriate animation when I click on it on-screen-wise but when I press down the corresponding keys on my keyboard, nothing happens.I am on a Mac with macOS Catalina.

**Comment by utkarsh1311** [comment]
Yes, I know. I haven't implemented that yet. You're free to add them if you want to.
